### PR TITLE
Feature/update lib

### PR DESCRIPTION
**Pull request by mapyo**
ref. https://github.com/Findy/react-findy-id/pull/34#issuecomment-550638678

@deraru I resend this pr.

### CODE REVIEW

**Comment by deraru** [comment]
Thank you! Please check the localization file in the new version of firebaseui.
Ref. https://github.com/Findy/react-findy-id#localization

**Comment by mapyo** [comment]
image

<details><summary>npm start log</summary>

```
$ npm start

> @findy/react-findy-id@2.5.0 start /Users/mapyo/ghq/github.com/Findy/react-findy-id
> webpack-dev-server --mode development

ℹ ｢wds｣: Project is running at http://localhost:3001/
ℹ ｢wds｣: webpack output is served from /
Browserslist: caniuse-lite is outdated. Please run next command `npm update caniuse-lite browserslist`
ℹ ｢wdm｣: Hash: 42382ce2d1506fdb6bd3
Version: webpack 4.25.1
Time: 4888ms
Built at: 11/13/2019 10:38:19 PM
       Asset       Size  Chunks             Chunk Names
./index.html  388 bytes          [emitted]
     main.js    2.3 MiB    main  [emitted]  main
Entrypoint main = main.js
[0] multi (webpack)-dev-server/client?http://localhost:3001 ./examples/src/index.js 40 bytes {main} [built]
[./examples/src/index.js] 3.67 KiB {main} [built]
[./node_modules/@babel/polyfill/lib/index.js] 893 bytes {main} [built]
[./node_modules/@babel/polyfill/node_modules/core-js/es6/index.js] 5.74 KiB {main} [built]
[./node_modules/loglevel/lib/loglevel.js] 7.68 KiB {main} [built]
[./node_modules/react-dom/index.js] 1.33 KiB {main} [built]
[./node_modules/react/index.js] 190 bytes {main} [built]
[./node_modules/strip-ansi/index.js] 161 bytes {main} [built]
[./node_modules/url/url.js] 22.8 KiB {main} [built]
[./node_modules/webpack-dev-server/client/index.js?http://localhost:3001] (webpack)-dev-server/client?http://localhost:3001 7.78 KiB {main} [built]
[./node_modules/webpack-dev-server/client/overlay.js] (webpack)-dev-server/client/overlay.js 3.58 KiB {main} [built]
[./node_modules/webpack-dev-server/client/socket.js] (webpack)-dev-server/client/socket.js 1.05 KiB {main} [built]
[./node_modules/webpack/hot sync ^\.\/log$] (webpack)/hot sync nonrecursive ^\.\/log$ 170 bytes {main} [built]
[./node_modules/webpack/hot/emitter.js] (webpack)/hot/emitter.js 75 bytes {main} [built]
[./src/index.js] 407 bytes {main} [built]
    + 320 hidden modules
Child html-webpack-plugin for "index.html":
     1 asset
    Entrypoint undefined = ./index.html
    [./node_modules/html-webpack-plugin/lib/loader.js!./examples/src/index.html] 545 bytes {0} [built]
    [./node_modules/lodash/lodash.js] 528 KiB {0} [built]
    [./node_modules/webpack/buildin/global.js] (webpack)/buildin/global.js 489 bytes {0} [built]
    [./node_modules/webpack/buildin/module.js] (webpack)/buildin/module.js 497 bytes {0} [built]
ℹ ｢wdm｣: Compiled successfully.
```

</details>


@deraru 
I checked the localization file.

Is this check OK?

**Comment by deraru** [comment]
No, it isn't. You need build a new localization file on local machine, and update the localization file in this library if there are any diff.

**Comment by mapyo** [comment]
Oh.. I misunderstood `Run on local example app` .
I will try `localization`.

**Comment by mapyo** [comment]
@deraru 
I updated the localization file, and `npm audit fix`.

File: README.md [hunk 1]
```diff
@@ -77,6 +77,6 @@ privacyPolicyUrl|String|Privacy Policy URL.
 - `cd firebaseui-web`
 - `npm install`
 - `npm run build build-npm-ja`
-- `cp dist/npm__ja.js <react-findy-id directory>/src`
+- `cp dist/npm__ja.js <react-findy-id directory>/lib`
```
Comment (deraru) [inline README.md]: 👍

**Pull request merged**